File: dehaze-tool/dehaze-sdk-android/src/main/java/com/pei/dehaze/sdk/model/dict/DictForm.java
package com.pei.dehaze.sdk.model.dict;

import lombok.Data;

/**
 * 字典表单模型类
 */
@Data
public class DictForm {
    /**
     * 字典ID
     */
    private Integer id;
    
    /**
     * 字典名称
     */
    private String name;
    
    /**
     * 排序
     */
    private Integer sort;
    
    /**
     * 状态(1:启用;0:禁用)
     */
    private Integer status;
    
    /**
     * 类型编码
     */
    private String typeCode;
    
    /**
     * 值
     */
    private String value;
    
    /**
     * 备注
     */
    private String remark;
}

File: dehaze-go/model/bo/dict_form.go
package bo

// DictFormBO 字典表单对象
type DictFormBO struct {
	// 字典ID
	ID *int64 `json:"id"`
	// 类型编码
	TypeCode string `json:"typeCode"`
	// 字典名称
	Name string `json:"name"`
	// 字典值
	Value string `json:"value"`
	// 状态(1:启用;0:禁用)
	Status int8 `json:"status"`
	// 排序
	Sort int `json:"sort"`
	// 字典备注
	Remark string `json:"remark"`
}
